packages/dds/merge-tree/src/collections.ts

```diff
@@ -467,6 +467,64 @@ export class RedBlackTree<TKey, TData> implements Base.SortedDictionary<TKey, TD
         }
     }
 
+    walkExactMatchesForward(
```

Comment:
would be good to add some ut converage for new methods

Reply:
Sounds good. Is there a specific UT I should add to?

Reply:
not specifically. just exercise the new methods. doesn't need to be exhaustive coverage. goal is just to prevent regression if someone changes something

Reply:
Okay. The iteration testing in this PR does exercise the methods. Are you saying you'd like to see a UT all them explicitly?